package.json

```diff
@@ -244,7 +244,8 @@
     "elliptic": "6.6.1",
     "graphql-request": "^6.1.0",
     "json5": "^2.2.2",
-    "follow-redirects": "^1.15.4"
+    "follow-redirects": "^1.15.4",
+    "pbkdf2": "^3.1.3"
```

Comment:
It looks like we can revert changes in `package.json` and `yarn.lock` harmlessly